db/schema.rb

```diff
@@ -10,11 +10,41 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_12_132525) do
+ActiveRecord::Schema.define(version: 2020_04_15_020109) do
```

Comment:
Guessing you don't want these changes on this branch?

Reply:
Yup, that was a `git add -a` accident